### PR TITLE
[SPARK-29601][WEBUI] JDBC ODBC Tab Statement column provide ellipsis for big SQL statement

### PR DESCRIPTION
**Pull request by PavithraRamachandran**
### What changes were proposed in this pull request?
Provide Ellipses in Statement column , just like description in Jobs page . 

### Why are the changes needed?
When a query is executed the whole query statement is displayed no matter how big it is. When bigger queries are executed, it covers a large portion of the page display, when we have multiple queries it is difficult to scroll down to view all.

### Does this PR introduce any user-facing change?
No

Before:
Screenshot from 2019-11-01 23-15-23

After:
Screenshot from 2019-11-02 07-07-21

### How was this patch tested?
Manual

### CODE REVIEW

**Comment by PavithraRamachandran** [comment]
@srowen @wangyum  Could you review this PR?

**Review by srowen — CHANGES_REQUESTED** [review]

**Comment by PavithraRamachandran** [comment]
@srowen i have done the changes.Could you review?

File: sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala [hunk 1]
```diff
@@ -347,7 +347,9 @@ private[ui] class SqlStatsPagedTable(
         {formatDurationVerbose(duration)}
       </td>
       <td>
-        {info.statement}
+        <span class="description-input">
```
Comment (juliuszsompolski) [inline sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala]: does this ellipsis expand to show the full query when clicked on?
is it possible to add an "expand all" functionality?

Comment (PavithraRamachandran) [reply sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala]: yes it shows full query when clicked. 
In jobs page , stage page also similar ellipsis is present. But we don't have expand all in those pages too.
@srowen could you suggest if expand all needs to added in all the places for consistency?

Comment (srowen) [reply sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala]: I'd just match any similar other page. If there isn't an expand all on the jobs page, we don't need to add it here.

Comment (juliuszsompolski) [reply sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala]: I sometimes find it useful to be able to copy-paste the whole table with all query history out to a spreadsheet. Expand all would be useful for that. But I agree that it's not a core feature for this PR, and it's consistent with other tabs.
Maybe raise a followup JIRA with the idea to add an expand all like that to all tables of the UI?

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**